packages/webpack-plugin-component-config/README.md

```
@@ -2,9 +2,12 @@
 
 一个用于 UniApp 项目的 Webpack 插件，用于处理 Vue 文件中的 `<component-config>` 标签，将配置提取并合并到对应的 `小程序 JSON 文件` 中。
 
-> [!TIP]
+> [!WARNING]
 > **Node.js** >= 20.19.0
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)